app/manifest.go

```diff
@@ -173,6 +179,17 @@ type KindPermission struct {
 	Actions  []KindPermissionAction `json:"actions,omitempty" yaml:"actions,omitempty"`
 }
 
+type ManifestOperatorInfo struct {
```

Comment:
nit: let's add some godoc on this and the struct below as they're exported

Reply:
Ah, yeah, I should have done that in a prior PR, glad the moving of this section brought this up. Will do.